src/Path/UploadListings.jsx

```diff
@@ -128,23 +131,60 @@ function UploadListings() {
 
   const [localImages, setLocalImages] = useState([]);
   const handleMultipleInputChange = (event) => {
-    const newImages = Array.from(event.target.files);
-    setLocalImages((prevImages) => [...prevImages, ...newImages]);
-    setImage([...image, ...newImages]);
+    const newFiles = Array.from(event.target.files);
+
+    if (image.length > 0) {
+      const validImages = newFiles.filter(file => file.type.startsWith("image/"));
+
+      if (validImages.length > 0) {
+        setLocalImages((prevImages) => [...prevImages, ...validImages]);
+        setImage((prevImages) => [...prevImages, ...validImages]);
+      } else {
+        alert(t("imagePdfAlert"));
+      }
+    } else {
+      newFiles.forEach(file => {
```

Comment:
can do the above mentioned validation here too

Reply:
see if we are handling if the user uploads images and pdfs together

Reply:
Done